templates/github/.github/workflows/scripts/script.sh.j2

```diff
@@ -37,6 +37,12 @@ if [[ "$TEST" = "docs" ]]; then
   exit
 fi
 
+if [[ "$TEST" = "new_docs" ]]; then
```

Comment:
It's probably easier to just run this with the other "docs" test runner.
And in the long run we only want this one as "docs" anyway.